.travis.yml

```diff
@@ -81,6 +81,11 @@ script:
 after_success:
 - bash <(curl -s https://codecov.io/bash)
 - if [[ $TRAVIS_BRANCH == master && $TEST_TYPE == integration ]]; then ./gradlew uploadArchives; fi
+# if this is actually a commit to master and not a pull request build into master, then publish master-snapshot
+- if [[ $TRAVIS_BRANCH == master && $TRAVIS_PULL_REQUEST == false && $TEST_TYPE == integration ]]; then 
+    git tag master;
```

Comment:
@lbergelson Is it the case that line 83 already does this - it's just that it uses the commit-derived name ? So will this double upload for every merged PR ? Trying to understand the conditions.